File: TransCoder_tokenized_test_set_functions/java/COUNT_SUBSETS_DISTINCT_EVEN_NUMBERS.java
static int countSubsets ( int arr [ ], int n ) {
  HashSet < Integer > us = new HashSet < > ( );
  int even_count = 0;
  for ( int i = 0;
  i < n;
  i ++ ) if ( arr [ i ] % 2 == 0 ) us . add ( arr [ i ] );
  even_count = us . size ( );
  return ( int ) ( Math . pow ( 2, even_count ) - 1 );
}

File: results/py/COUNT_SUBSETS_DISTINCT_EVEN_NUMBERS.cpp
int f_filled ( int arr [ ] , int n ) {
  set < int > us ;
  int even_count = 0 ;
  for ( int i = 0 ;
  i < n ;
  i ++ ) {
    if ( arr [ i ] % 2 == 0 ) us . insert ( arr [ i ] ) ;
  }
  for ( int i = 0 ;
  i < us . size ( ) ;
  i ++ ) even_count ++ ;
  return pow ( 2 , even_count ) - 1 ;
}
